runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -3601,7 +3625,17 @@ void init_fixedHeap(void) {
                      chpl_snprintf_KMG_z(buf, sizeof(buf), size), size);
         }
 #endif
+      if (!useTHP) {
         CHK_SYS_MEMALIGN(start, page_size, size);
+      } else {
+        start = mmap(NULL, size,
```

Comment:
I suggest making a `CHK_SYS_MMAP` macro to be consistent with wrapping other memory allocations in macros.